recipes/cms/meta.yaml

```diff
@@ -0,0 +1,119 @@
+package:
+  name: cms
+  version: "2.0.0b"
+
+about:
+  home: http://broad-cms.readthedocs.io/en/latest/
+  license: GPLv3
+  summary: "CMS (composite of multiple signals) is a suite of tools for resolving signatures of recent positive selection in population genetic data."
+
+source:
+  git_url: https://github.com/broadinstitute/cms.git
+  git_rev: ee7efbb5b18697fc9bf2e813fe76fbbc2439a0de
```

Comment:
When there is a [CMS release](https://github.com/broadinstitute/cms/releases) you're ready to distribute, it would be good to use the release zip file (or .tar.gz)as the recipe source rather than a git revision, since the later imposes more overhead during the build process.